### PR TITLE
[SEO]: improve h1 tags. 

### PR DESCRIPTION
**Pull request by nisarhassan12**
Addresses parts of #1681.

- [x] fix the "h1 tags being too short" suggestions from Christin in the [document](https://gitpod.slack.com/files/U01KVKJJ3J8/F035DSKSXGC/_h1__length_too_short) are implemented the rest will hopefully be addressed in future iterations. 
- [x]  multiple h1 tags
- [x] missing h1 tag fixed in https://github.com/gitpod-io/website/pull/1712/commits/fc6df7a532db19a0b14144f4548a020d239a17a8

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1712"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1712"></a>

### CODE REVIEW

File: src/routes/changelog/index.svelte [hunk 1]
```diff
@@ -54,7 +54,7 @@
   <div class="hidden w-4/12 flex-shrink-0 md:block" />
   <Header
     title="Changelog"
-    text="Gitpod product improvements and updates"
+    text="A sum-up of Gitpod’s latest product improvements, feature releases and community contributions."
```
Comment (ChristinFrohne) [inline src/routes/changelog/index.svelte:57]: Can you break the text into the next line for desktop view after "releases"?
Bildschirmfoto 2022-03-07 um 09 23 09

File: src/lib/components/blog/digest-header.svelte [hunk 2]
```diff
@@ -1,19 +1,25 @@
 <script lang="ts">
-  import Header from "../header.svelte";
   import TwitterFollowButton from "../t-button.svelte";
   import LinkButton from "$lib/components/ui-library/link-button";
 </script>
 
-<Header
-  title="DevX Digest"
```
Comment (ChristinFrohne) [inline src/lib/components/blog/digest-header.svelte:8]: Can you add a label above the headline? By that we can differentiate is more from the usual blog posts
Bildschirmfoto 2022-03-07 um 09 22 03

**Review by ChristinFrohne — COMMENTED** [review]
Thanks!
I added a few comments inline.
regarding missing h1 tag -> [affected pages](https://docs.google.com/spreadsheets/d/1GjNlJ8wpKW5Guo5xdNsXbT9DggE2c44b2VfKSXeGxOs/edit#gid=0), I think we have already deindexed all single Changelog entries. Is that correct? Then that issue would be fixed as well.

**Comment by nisarhassan12** [comment]
@ChristinFrohne Thanks for the review I have made the requested changes.

> I think we have already deindexed all single Changelog entries. Is that correct?

No, they are not de-indexed at the moment. I have updated the PR to do so. They'll be once this PR is merged.

**Comment by ChristinFrohne** [comment]
>No, they are not de-indexed at the moment. I have updated the PR to do so. They'll be once this PR is merged.

Aren't they already deindexed in this PR? https://github.com/gitpod-io/website/pull/1604 cc @Cahllagerfeld 

Otherwise the PR LGTM :)

**Comment by Cahllagerfeld** [comment]
Exactly @ChristinFrohne, with the marked PR a `layout` gets added that adds `<meta name="robots" content="noindex" />` to the detail changelog-pages

**Comment by nisarhassan12** [comment]
> Exactly @ChristinFrohne, with the marked PR a layout gets added 

Sorry, @Cahllagerfeld I didn't know about this and I was not able to figure this out as I was unaware that we also use a layout file just to do that. 

Let's go with what is being used more often for consistency. I'll update this Pr to get rid of the `_layout.svelte` file.

**Pull request merged**